crates/gpui/src/platform/windows/window.rs

```diff
@@ -258,7 +261,11 @@ impl WindowsWindowInner {
         LRESULT(0)
     }
 
-    fn handle_size_msg(&self, lparam: LPARAM) -> LRESULT {
+    // todo(windows): if maximized, not to record window size to DB
+    fn handle_size_msg(&self, wparam: WPARAM, lparam: LPARAM) -> LRESULT {
+        if wparam.0 as u32 == SIZE_MINIMIZED {
```

Comment:
I don't think this is the correct solution to this problem.

I think it would be better to faithfully reflect the platform's state, so taking out this check, and then add a `WindowContext::is_minimized` function, which checks this state from the platform window. Then, in the workspace serialization, specifically `crates/workspace/src/workspace.rs` on line 757, we can expand that line from an `} else {` to `} else if !cx.is_minimized() {`. 

Feel free to leave the other platform implementations for me to pick up.

Reply:
I added `is_minimized`, so please take over.